java/trace/src/test/scala/org/typelevel/otel4s/java/trace/TracerSuite.scala

```diff
@@ -19,7 +19,10 @@ package org.typelevel.otel4s.java.trace
 import cats.effect.IO
 import cats.effect.IOLocal
 import cats.effect.Resource
+import cats.effect.kernel.MonadCancelThrow
 import cats.effect.testkit.TestControl
+import cats.implicits.toFunctorOps
```

Comment:
I prefer `import cats.syntax.X._` over `import cats.implicit.X`, but it's a matter of taste :)

I wonder, if we agree to use a specific style (e.g. `import cats.syntax.X._`) project wide, can we enforce it via scalafix?

Reply:
I am fine with that - I believe this was IntelliJ's automatic choice

Reply:
If we enforce it (I like consistency), that sounds like it should be submitted as a [typelevel scalafix](https://github.com/typelevel/typelevel-scalafix).